.github/workflows/check-pep8.yml

```diff
@@ -2,7 +2,8 @@ name: autopep8
 on:
   push:
     branches:
-      - "*"
+      - main
```

Comment:
For the other two files you have the branch names in quotes, but here not. Do we need the double quotes? If not, lets keep things clean :)

Reply:
> The characters *, [, and ! are special characters in YAML. If you start a pattern with *, [, or !, you must enclose the pattern in quotes.

From the [github actions documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet). So sadly we need the quotes.

Reply:
Okay, then lets also have quotes in the check-pep8 workflow file.